profile-service/src/main/resources/db/migration/V9.14__update_reval_role_name.sql

```diff
@@ -0,0 +1,30 @@
+-- Get rid of the existing roles and detach them from the users
+
+DELETE
+FROM `UserRole`
+WHERE `roleName`
+  IN
+  ('RevalSiteAdmin',
+	'RevalSuperAdmin',
+	'RevalTISAdmin1',
+	'RevalTISAdmin2',
+	'RevalTISAdmin3');
```

Comment:
Do we really want to `DELETE`? or we can do `UPDATE` to change old roleName to new roleName instead?

Reply:
I am checking that option. If that works then it will eleminate the necessity of migration. :+1: